cl/people_db/tasks.py

```diff
@@ -22,37 +25,28 @@ def make_png_thumbnail_for_instance(
     :param max_dimension: The longest you want any edge to be
     """
     item = InstanceClass.objects.get(pk=pk)
-    command = [
-        "pdftoppm",
-        "-jpeg",
-        getattr(item, file_attr).path,
-        # Start and end on the first page
-        "-f",
-        "1",
-        "-l",
-        "1",
-        # Set the max dimension (generally the height). Alas, we can't just
-        # set the width, so this is our only hope.
-        "-scale-to",
-        str(max_dimension),
-    ]
-
-    # Note that pdftoppm adds things like -01.png to the end of whatever
-    # filename you give it, which makes using a temp file difficult. But,
-    # if you don't give it an output file, it'll send the result to stdout,
-    # so that's why we are capturing it here.
-    p = subprocess.Popen(
-        command, close_fds=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE
-    )
-    stdout, stderr = p.communicate()
-    if p.returncode != 0:
-        item.thumbnail_status = THUMBNAIL_STATUSES.FAILED
-        item.save()
-        return item.pk
+    filepath = getattr(item, file_attr).path
+
+    try:
+        thumbnail_resp = generate_thumbnail(filepath)
+    except Timeout:
+        logger.error("Thumbnail generation failed via timeout.")
+    except Exception as e:
+        logger.error(
+            "Catch all exception occurred during thumbnail generation.  See %s"
+            % str(e)
+        )
```

Comment:
You might as well just not have this catch all. WIthout it, an error goes to Sentry with full stacktrace and other great stuff. With it, we only get the log line.

```suggestion
```